app/models/dqt_record.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: dqt_records
+#
+#  id                   :bigint           not null, primary key
+#  date_of_birth        :date
+#  early_years_status   :string
+#  eyts_date            :date
+#  first_name           :string
+#  has_active_sanctions :boolean
+#  last_name            :string
+#  ni_number            :string
+#  qts_date             :date
+#  trn                  :string
+#  uuid                 :string
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#
+class DqtRecord < ApplicationRecord
```

Comment:
Why are we storing these in the database?

Reply:
I assumed this was what was meant when the ticket said associate the record?

Reply:
The two acceptance criteria are:

- When users click Add record on the user page, we immediately look up Kevin E's TRN, but ask them to confirm before we associate it with the record
- The page looks like the design and is populated with data both from the Identity API as well as the `qualified-teachers-api`

Find shouldn't persist need to persist anything to associate a record.

Find's receives a user's data from Get an Identity, and displays it. Then we ask the user for a TRN to look up on the DQT API. The DQT API returns a record, and then we ask the user to confirm that they want to associate the Get an Identity record with that DQT record.